docs/schemas/schema-spat.json

```diff
@@ -22,6 +23,9 @@
                 "sanitized"
             ],
             "properties": {
+                "@class": {
```

Comment:
Not sure how these JSON schemas are related to this doc: https://github.com/usdot-jpo-ode/jpo-ode/blob/dev/docs/ODE_Output_Schema_Reference.docx? Do these schema.json changes require the change in the doc as well to make sure they are both updated?

Reply:
Yeah, that document could probably be updated. Realistically adding a description to every field is not the goal of this pull request though. This is a JSON schema update for validation purposes where that document is more for human readability and understanding. I think it would be reasonable to add to that documentation for consistency's sake but I do not have this done.

Reply:
How to use this JSON schema file to validate the output? Maybe add a readme.md under this folder to describe how to use this JSON schema file?

Reply:
At second glance, that document is a Java class document. The schema updates being made don't affect the Java code. A '@class' field has been added to the JSON output that is absent from the Java classes. This realistically should have been included in pull request #486 but it wasn't so I am doing it.

Reply:
I see. Sorry for the confusion. My comment is about the JSON files usage, not a particular question to a specific update like @class field.

Reply:
Okay, I can add a readme.

Reply:
Okay, I have added a readme for the schemas. Let me know what you think.